tests/testthat/helper-paths.R

```diff
@@ -0,0 +1,45 @@
+pythonPathOrSkip <- function() {
+  skip_if_not_installed("reticulate")
+
+  path <- getPython()
```

Comment:
Not sure if it's meaningful, but the existing tests skipped when `Sys.which` indicated that there was no Python.. and then used that Python to check for `pip`. That Python was then used to set the environment variables used internally by `getPython`.

The `getPython` function relies on environment variables.

Reply:
Ok, I'll contemplate that.

Reply:
In hindsight, this is an easy fix — just always rely on reticulate to find python. I'll double check that it works on CI, but I think this is the right approach for tests.